shotover-proxy/tests/runner/observability_int_tests.rs

```diff
@@ -26,16 +90,31 @@ async fn test_metrics() {
         .await
         .unwrap();
 
-    let client = hyper::Client::new();
-    let uri = "http://localhost:9001/metrics".parse().unwrap();
-    let res = client.get(uri).await.unwrap();
-    let body_bytes = hyper::body::to_bytes(res.into_body()).await.unwrap();
-    let body = String::from_utf8(body_bytes.to_vec()).unwrap();
-
-    // If the body contains these substrings, we can assume metrics are working
-    assert!(body.contains("# TYPE shotover_transform_total counter"));
-    assert!(body.contains("# TYPE shotover_chain_total counter"));
-    assert!(body.contains("# TYPE shotover_available_connections gauge"));
-    assert!(body.contains("# TYPE shotover_transform_latency summary"));
-    assert!(body.contains("# TYPE shotover_chain_latency summary"));
+    redis::cmd("GET")
+        .arg("the_key")
+        .query_async::<_, ()>(&mut connection)
+        .await
+        .unwrap();
+
+    std::thread::sleep(time::Duration::from_secs(1));
+    body = http_request_metrics().await;
+
+    let new_lines = vec![
```

Comment:
Please add a TODO comment explaining that we dont actually assert on the contents of these yet, due to the randomized order of the labels but a proper solution needs to be investigated.
As is, this is very misleading!